modular_skyrat/modules/cortical_borer/code/cortical_borer.dm

```diff
@@ -161,15 +161,20 @@ GLOBAL_VAR_INIT(successful_blood_chem, 0)
 										/datum/reagent/medicine/c2/lenturi,
 										/datum/reagent/medicine/c2/convermol,
 										/datum/reagent/medicine/c2/seiver,
+										/datum/reagent/medicine/c2/multiver,
 										/datum/reagent/lithium,
 										/datum/reagent/medicine/salglu_solution,
 										/datum/reagent/medicine/mutadone,
 										/datum/reagent/toxin/heparin,
-										/datum/reagent/medicine/mannitol,
 										/datum/reagent/drug/methamphetamine/borer_version,
 										/datum/reagent/medicine/morphine,
 										/datum/reagent/medicine/inacusiate,
 										/datum/reagent/medicine/oculine,
+										/datum/reagent/toxin/mindbreaker,
+	)
+	//blacklisted chemicals - separate from chemicals that cannot be synthesized, borers specifically cannot learn these
+	var/list/blacklisted_chemicals = list(/datum/reagent/medicine/mannitol,
+										/datum/reagent/medicine/neurine,
```

Comment:
I'm against creating a blacklist whatsoever-- I'll live and we can see how it goes.